src/graphql/types/object/business-account.ts

```diff
@@ -7,12 +7,11 @@ import Transaction from "./transaction"
 import WalletId from "../scalar/wallet-id"
 
 import * as Wallets from "@app/wallets"
-import * as Accounts from "@app/accounts"
 
 const BusinessAccount = new GT.Object({
   name: "BusinessAccount",
   interfaces: () => [IAccount],
-  isTypeOf: (source) => source.title || source.coordinates, // TODO: improve
+  // isTypeOf: (source) => source.title || source.coordinates, // TODO: improve
```

Comment:
Can we delete this and document a TODO as GitHub issue instead of keeping commented code?

Reply:
IMO it's a better practice to document in code as opposed to issue. it's contextualized. 

but actually I should take care of this one before merging. I thought BusinessAccount were not used (I had businessAccount as corporate account in my mind, but they are consumer account with maps metadata), but they actually are already.

Reply:
> IMO it's a better practice to document in code as opposed to issue. it's contextualized.
Something to gain team consensus on I guess

Reply:
we could discuss it during code review, my assumption is that is already part of the consensus, the code has ~200 FIXME and TODO.